### ISSUE TITLE
In python3.5 and opencv 3.1, I got a error

### ISSUE THREAD

**Issue by CarGod** [VasuAgrawal/GestureDetection#1]
Thanks Vasu Agrawal.
First,This is a very good project!
But in python3.5 and opencv3.1.0 I got a error.
Error manage as follows:
ValueError: too many values to unpack (expected 2)
`
GesturesApi.py", line 122
cv2.CHAIN_APPROX_SIMPLE)
`

**Comment by VasuAgrawal** [comment]
Do you need to use Python 3.5 and OpenCV 3.1.0? If not, please consider using Python 2 and OpenCV 2 as that is the tested configuration.

I suspect that the OpenCV API has changed slightly from 2 to 3, which is what's causing the unpacking issue. Switching the entire codebase from 2 to 3 is a nontrivial task.

**Comment by CarGod** [comment]
No. I don't, In python2 and opencv2 is ok. I would like to try it in python3 and opencv3.
